test/FunctionalTests/Tests/DataEdmLib/FunctionalTests/VocabularyParsingTests.cs

```diff
@@ -1726,16 +1726,16 @@ public void EnumAnnotationParsingTest()
   <Term Name=""Permission"" Type=""Ns.Permissions"" Nullable=""false"" />
   <EntityContainer Name=""Container"">
     <EntitySet Name=""People"" EntityType=""Ns.Person"">
-        <Annotation Term=""Org.OData.Capabilities.V1.NavigationRestrictions"">
+        <Annotation Term=""Test.NavigationRestrictions"">

```

Comment:
This case is intended to test the undefined vocabulary. Before this feature, `Org.OData.Capabilities.V1.NavigationRestrictions` is an undefined vocabulary. After this feature, it now becomes a defined vocabulary in the Capabilities vocabulary model. Therefore, we should change the input for this test case. As suggestion, changed to "Test" is acceptable.